src/atopile/front_end.py

```diff
@@ -100,6 +103,18 @@ def tolerance_pct(self) -> float:
     def __repr__(self) -> str:
         return f"<{self.__class__.__name__} {self.min_val} to {self.max_val} {self.unit}>"
 
+    def to_json(self) -> dict:
```

Comment:
nit; this should be named `to_dict` since it's returning a dict, not a JSON string.